### ISSUE TITLE
AVT 1 - Tag (default & filter) React Component has DAP violations 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4428]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon React Version 7.7.1
DAP 1.8.0.0 Sept 2019 WCAG 2.1 ruleset

## Description
Run DAP on the Tag component and the following violation is present:
<img width="1142" alt="Screen Shot 2019-10-22 at 2 44 38 PM" src="https://user-images.githubusercontent.com/21676914/67325734-38d4dc00-f4db-11e9-8ab4-e7c688b66972.png">
